data/pirate missions smuggling intro.txt

```diff
@@ -0,0 +1,1408 @@
+			label questions
+			choice
+				`	"Five deliveries for 500K? Sounds like a deal to me."`
+					goto yes
+				`	"Anything for you, angel."`
+					goto angel
+				`	"On the hook? What happens if I only do a couple?"`
+					goto hook
+				`	"What would I be hauling?"`
+					goto hauling
+				`	"When can I expect my first job?"`
+					goto when
+				`	"Do I need to go somewhere in particular?"`
+					goto somewhere
+				`	"Sorry, I'm not interested. I'll make my own way."`
+					goto no
+			label hook
+			`	She gives you a sad look. "Well, we'll keep asking you, but if you don't make the runs you've agreed to, we'll have to report you to the Republic for breaking contract and collect our 500,000. It should go without saying, but abandoning deliveries and missing deadlines is a big no-no. But you don't look like the type of person who would do that!"`
+				goto questions
+			label hauling
+			`	"TBD - simple goods and maybe passengers. Depends on what's needed at the time. Nothing too large, though."`
+				goto questions
+			label when
+			`	Alexandra looks at her datapad and taps the screen rapidly. "I'm guessing we'll have something for you within a couple weeks. We'll only call you when you're the closest courier, but the average wait time is about ten days after every run."`
+				goto questions
+			label somewhere
+			`	"Not really. Most of our work is in Republic space, and we would prefer it if you tried to stay close. But if you want to catch a few extra jobs here or there, feel free!"`
+				goto questions
+			
+			label yes
+			`	Alexandra smiles. "Great! I'll transfer the money to you right away, <first>. We'll be in touch. In the meantime, enjoy your new-found freedom!"`
+				goto meetup7
+			label angel
+			`	Alexandra laughs. "Oh, this is going to be fun. I'll transfer the money to you right away -  I'm looking forward to working with you, <first>. Enjoy your freedom!"`
+				goto meetup7
+			
+			label meetup7
+			`	When you make it back to your ship, you wonder if what Alexandra said is too good to be true; but sure enough, 500,000 credits have been transferred to your account; more money than you thought you'd have in a lifetime, and more than enough to pay off your debt.`
+				accept
+		
+			label no
+			`	Alexandra nods her head sadly. "Are you sure, <first>? I've heard horror stories. First-time pilots who suffer a few setbacks, being chased down by loan officers... I couldn't bear the thought."`
+			choice
+				`	"On second thought, being debt-free sounds awesome. I'll do it."`
+					goto yes
+				`	"Sorry, it's just not for me. I'll do it the old-fashioned way."`
+			label endno
+			`	Alexandra sighs, then smiles. "Alright! Well, thank you for your time. Perhaps we'll bump into each other again!"`
+				decline
+	
+	on accept
+		set "liekedeeler agreement"
+		event "Liekedeeler Delivery 1" 8 10
+		log "People" "Alexandra Merrin" `Alexandra Merrin is a representative of a courier firm named 'Liekedeeler Enterprises.'`

```

Comment:
`log "People" "Alexandra Merrin" `Alexandra Merrin is a representative of a courier firm named 'Liekedeeler Enterprises,' one that helps new pilots pay off their loans in exchange for a rather lucrative contract agreement.`
`